packages/stream_video_push_notification/ios/Classes/StreamVideoPKDelegateManager.swift

```diff
@@ -8,7 +8,7 @@ public class StreamVideoPKDelegateManager: NSObject, PKPushRegistryDelegate {
     
     private var pushRegistry: PKPushRegistry?
     private var defaultData: [String: Any]?
-    private var methodChannel: FlutterMethodChannel?
+    private var mainChannel: FlutterMethodChannel?
```

Comment:
What was the reason behind the name change?

Reply:
It was from when I tried to create separate `backgroundChannel`. If it's not a big problem I would leave the `mainChannel` name since we might need the `backgroundChannel` anyway in the future.